ui/components-react/src/components-react/tree/controlled/component/TreeNodeRenderer.tsx

```diff
@@ -70,6 +70,7 @@ export const TreeNodeRenderer = React.memo((props: TreeNodeRendererProps) => {
       nodeEditorRenderer={props.nodeEditorRenderer}
     />
   );
+  TreeNodeRenderer.displayName = "TreeNodeRenderer";
```

Comment:
Can you move this outside of `TreeNodeRenderer`? Now it looks like this is a part of component implementation.

Reply:
Yes, my bad.